[libzippp] Fix conditional check for WIN32 in portfile for vcpkg_fixup_cmake_targets

****

**Is your feature request related to a problem? Please describe.**
When building libzippp for Android on my Windows host, I get the following error because of the conditional on [line 21 of libzippp's portfile](https://github.com/microsoft/vcpkg/blob/master/ports/libzippp/portfile.cmake#L21):
```
CMake Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:72 (message):
  'C:/a/4/s/__vcpkg/packages/libzippp_arm-android/debug/cmake/libzippp' does
  not exist.
Call Stack (most recent call first):
  ports/libzippp/portfile.cmake:22 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:79 (include)

[DEBUG] cmd_execute_and_stream_data() returned 1 after  5080881 us
Error: Building package libzippp:arm-android failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libzippp:arm-android
  Vcpkg version: 2020.06.15-nohash
```

**Proposed solution**
Instead of checking `if(WIN32)`, the portfile should be checking `if(VCPKG_TARGET_IS_WINDOWS)`. Alternatively, it could check for existence of the folders/files in question to be more robust to future changes.

--- a/ports/libzippp/CONTROL
+++ b/ports/libzippp/CONTROL
@@ -1,5 +1,6 @@
 Source: libzippp
 Version: 4.0-1.7.3
+Port-Version: 1
 Homepage: https://github.com/ctabin/libzippp
 Description: Simple basic C++ wrapper around the libzip library. It is meant to be a portable and easy-to-use library for ZIP handling
 Build-Depends: zlib, libzip[bzip2]

--- a/ports/libzippp/CONTROL
+++ b/ports/libzippp/CONTROL
@@ -3,4 +3,4 @@ Version: 4.0-1.7.3
 Port-Version: 1
 Homepage: https://github.com/ctabin/libzippp
 Description: Simple basic C++ wrapper around the libzip library. It is meant to be a portable and easy-to-use library for ZIP handling
-Build-Depends: zlib, libzip[bzip2]
+Build-Depends: zlib, libzip[core,bzip2]

--- a/ports/libzippp/portfile.cmake
+++ b/ports/libzippp/portfile.cmake
@@ -1,4 +1,3 @@
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ctabin/libzippp
@@ -18,7 +17,7 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-if(WIN32)
+if(VCPKG_TARGET_IS_WINDOWS)
     vcpkg_fixup_cmake_targets(CONFIG_PATH "cmake/libzippp")
 else()
     vcpkg_fixup_cmake_targets(CONFIG_PATH "share/libzippp")
@@ -27,4 +26,4 @@ endif()
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENCE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libzippp RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENCE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

